server/controllers/finance/cash.js

```diff
@@ -35,25 +35,29 @@ exports.debitNote = debitNote;
 
 // looks up a single cash record and associated cash_items
 // sets the "canceled" flag if a cash_discard record exists.
-function lookupCashRecord(uuid, codes) {
+function lookupCashRecord(id, codes) {
```

Comment:
Just a note that this should be updated given https://github.com/IMA-WorldHealth/bhima-2.X/issues/182. This is not directly related to the pull request and will not block it. 

This should be addressed in the Cash 2.x Review.